### PR TITLE
LPS-145300 + LPS-145303 improve the notifications bulk actions

### PR DESCRIPTION
**Pull request by boton**
**Issues:**

[LPS-145303 Disable 'Notification no longer applies' row checker](https://issues.liferay.com/browse/LPS-145303)
[LPS-145300 Remove management toolbar restrictions on mark as read/unread actions](https://issues.liferay.com/browse/LPS-145300)

I fix these issues with @AliciaGarciaGarcia.

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java [hunk 1]
```diff
@@ -37,6 +38,14 @@ public boolean isDisabled(Object object) {
...
 		}
 
+		UserNotificationEvent storedUserNotificationEvent =
+			UserNotificationEventLocalServiceUtil.fetchUserNotificationEvent(
+				userNotificationEvent.getUserNotificationEventId());
+
+		if (storedUserNotificationEvent == null) {
+			return true;
+		}
+
```
Comment (adolfopa) [inline modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java:48]: Can you rename this to `userNotificationEvent`? Unless there's a reason to disambiguate, we usually use the raw type name as variable name.

**Review by adolfopa — APPROVED** [review]

File: modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java (hunk 1 above)
Comment (boton) [reply modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java:48]: We have another `userNotificationEvent` in some lines above (line 33-34 the link isn't working well). 
Do you have a better name in mind?

Comment (adolfopa) [reply modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/search/UserNotificationEventRowChecker.java:48]: I didn't see that :facepalm:

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**